kernel/src/loader/mod.rs

```diff
@@ -15,6 +15,9 @@ use memory_model::{GuestAddress, GuestMemory};
 use sys_util;
 
 #[allow(non_camel_case_types)]
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+// Add here any other architecture that uses as kernel image an ELF file.
+// aarch64 uses the Portable Executable (PE) format and does not contain any ELF headers.
```

Comment:
Do you mean by the Windows OS? AFAIK, aarch64 also use the ELF format on Linux OS.

Reply:
Indeed xiaobo, I was under some false impression. There is a indeed a vmlinux file (ELF format) obtained when compiling the aarch64 kernel, however in the current PR we are implementing the boot protocol for the standard format which is given by the arch/arm64/Image file and that is documented in the Documentation/arm64/booting.txt. I removed the comment as a consequence.

Reply:
Actually, the "MZ" signature in the image is required by UEFI, and it's not really mean it's a MS-DOS executable. In Linux, we usually use the ELF format for the kernel image, and PE on used in Windows OS.